pkg/bucketeer/user.go

```diff
@@ -31,3 +32,13 @@ func (u *User) Valid() bool {
 	}
 	return true
 }
+
+// NewUser creates a new User.
+//
+// id is mandatory and attributes is optional.
+func NewRestUser(id string, attributes map[string]string) *user.User {
```

Comment:
Because we have only one project using go-server-sdk, and also I have already talked to the engineer about the changes, there is no need to rename it. Let's keep it as it is.

Reply:
Talked at slack.